package.json

```diff
@@ -219,11 +219,11 @@
     }
   },
   "dependencies": {
-    "@patternfly/elements": "^2.4.0",
+    "@patternfly/elements": "^3.0.0",
```

Comment:
I want this to be a dev dependency, or to be removed entirely

We should only need core and tools

Reply:
on second thought, i have to revise my previous comment. we still rely on `<pf-icon>` e.g. in footer, so for now we'll keep the dependency, but the only imports should be pf-icon, really